src/cljs/rems/config.cljs

```diff
@@ -24,3 +27,39 @@
 (defn dev-environment? []
   (let [config @(rf/subscribe [::config])]
     (boolean (:dev config))))
+
+;; organizations
```

Comment:
should we also move fetching themes and translations to this file?

Reply:
Could be but maybe it also should be named after the APIs i.e. public? Or is that wrong too? Organizations are not exactly config so they could be in their own ns. They are also not public.